examples/SimpleTxtPlugin/SimpleTxtPlugin.csproj

```diff
@@ -5,12 +5,12 @@
     <Nullable>enable</Nullable>
     <EnableDynamicLoading>true</EnableDynamicLoading>
     <GeneratePluginPackageOnBuild>true</GeneratePluginPackageOnBuild>
-    <RootNamespace>Zeiss.FirstImportFormat</RootNamespace>
+    <RootNamespace>SimpleTxtPlugin</RootNamespace>
```

Comment:
We could even remove `RootNamespace` and `AssemblyName` here, because by default those properties will already be set to the current project name which is `SimpleTxtPlugin`.

Reply:
Let's remove this completely then.